python/ray/dag/compiled_dag_node.py

```diff
@@ -94,47 +102,26 @@ def do_exec_tasks(
         tasks: the executable tasks corresponding to the actor methods.
```

Comment:
TODO: add comments for `schedule`.

Reply:
The method docstring needs to be updated as well.

Reply:
good catch

Reply:
Updated https://github.com/ray-project/ray/pull/46911/commits/5c3750ed58716da39fb2b93e17b9af225c9a0fee